releasenotes/notes/clifford_cz_lnn_synthesis-4b0328b581749df5.yaml

```diff
@@ -0,0 +1,17 @@
+---
+features:
+  - |
+    Add a new synthesis algorithm `.synth_cz_depth_line_mr` of a CZ circuit
+    for linear nearest neighbor (LNN) connectivity in 2-qubit depth of 2n+2
+    using CX and phase gates (S, Sdg or Z). The synthesized circuit reverts
+    the order of the qubits. The synthesis algorithm is based on the paper of Maslov and Roetteler
+    (https://arxiv.org/abs/1705.09176).
+    Add a new synthesis algorithm `.synth_clifford_depth_lnn` of a Clifford circuit
```

Comment:
Maybe it's worth to list this as a separate feature (i.e. after another `- |`). Also please give it a quick read-through, there are a few places where you may want to change "synthesize" to "synthesizes", etc.